src/commands/watch.js

```diff
@@ -19,8 +19,7 @@ export default asyncCommand({
 		},
 		host: {
 			description: 'Hostname to start a server on',
-			default: '0.0.0.0',
-			alias: 'h'
```

Comment:
Please move this to a seperate PR as your PR could be squashed to create a clean git history.